src/components/List/List.tsx

```diff
@@ -61,23 +52,34 @@ export const List = <Item extends Entity>(props: ListProps<Item>) => {
     LoadMoreButtonProps,
     scrollRef: scrollRefProp,
     className,
+    sx,
+    containerSx,
   } = props;
-  const { classes, cx } = useStyles(props, {
-    props: { classes: props.classes },
-  });
 
   const scrollRef = useRef<HTMLDivElement | null>(null);
   usePersistedScroll(scrollRefProp ?? scrollRef);
   const determineChangesetDiff = useDetermineChangesetDiffItem();
 
   return (
-    <div className={cx(classes.root, className)} ref={scrollRef}>
+    <Box
+      component="div"
+      className={className}
+      ref={scrollRef}
+      css={(theme) => [
+        {
+          overflow: 'auto',
+          marginLeft: theme.spacing(-2),
+          padding: theme.spacing(2),
+        },
+      ]}
+      sx={containerSx}
```

Comment:
Let's stick with the single declaration so we can be sure which css values are applied in which order. Since we have `sx` here, let's use it. I have a helper for this case.
```ts
sx={[
   { overflow: 'auto', ml: -2, p: 2 },
   ...extendSx(containerSx),
]}
```
Since `containerSx` is listed after definitions, it'll override our defaults.